balancer-js/examples/pools/composable-stable/join.ts

```diff
@@ -0,0 +1,99 @@
+import dotenv from "dotenv";
+import { Network } from "@/lib/constants";
+import { BalancerSDK } from "@/modules/sdk.module";
+import { ethers } from "hardhat";
+import pools_14717479 from "@/test/lib/pools_14717479.json";
+import { Pool } from "@/types";
+import { parseFixed } from "@ethersproject/bignumber";
+import { Pools } from "@/modules/pools";
+import { forkSetup, getBalances } from "@/test/lib/utils";
+
+dotenv.config();
+
+const { ALCHEMY_URL: jsonRpcUrl } = process.env;
+const rpcUrl = 'http://127.0.0.1:8545';
+const network = Network.MAINNET;
+const blockNumber = 16350000;
+
+const poolObj = pools_14717479.find(
+  ({ id }) =>
+    id == '0xa13a9247ea42d743238089903570127dda72fe4400000000000000000000035d'
+) as unknown as Pool;
+
+const sdk = new BalancerSDK({ network, rpcUrl });
+
+const { networkConfig } = sdk;
+
+const initialBalance = '100000';
+
+const tokensIn = [
```

Comment:
Just add a comment here that we're filtering out BPT token as new users might not know about that.